TcCommando/TcCommando/TcCommando/POUs/PLCOpenCall.TcPOU

```diff
@@ -233,12 +233,12 @@ END_VAR
         <ST><![CDATA[// Copyright (c) 2024 Loupe (https://loupe.team), provided under the MIT License.
 
 //Abort the previous command if it exists
-IF THIS^.Command <> Command THEN
+IF THIS^.CommandInternal <> Command THEN
```

Comment:
Considered renaming the input to this method (Command) however that would require changing the PLCopen interface signature. This seemed cleaner but I'm not convinced it's the most elegant.